docs/Docker-Support.md

```diff
@@ -79,6 +79,25 @@ ssh:
   auth_methods: ["password"]
 ```
 
+### Mounting volumes into your docker container ###
+You can mount folders into a docker container:
+
+    HOSTS:
+      ubuntu-12-10:
+        platform: ubuntu-12.10-x64
+        image: ubuntu:12.10
+        hypervisor: docker
+        mount_folders:
+          name1:
+            from: host_path1
```

Comment:
Maybe call from/to host_path/container_path ?
makes it a bit more verbose but a lot clearer.